htdocs/bom/class/bom.class.php

```diff
@@ -1324,7 +1323,7 @@ public function generateDocument($modele, $outputlangs, $hidedetails = 0, $hided
 	}
 
 	// phpcs:disable PEAR.NamingConventions.ValidFunctionName.ScopeNotCamelCaps
-	/**
+	 /**
```

Comment:
I am surprised about the spacing.

Reply:
I didn't change that one ;) I'm guessing the comment above confused it